test/third-party/miniflare.test.ts

```diff
@@ -0,0 +1,54 @@
+// @vitest-environment miniflare
```

Comment:
let me know if this test placement makes sense in `third-party`.  It's mostly an integration test for miniflare compat, which seems reasonable here

Reply:
Yes, this is the perfect place to put this. Thanks!